azure/azure-explanation/security-overview.rst

```diff
@@ -0,0 +1,109 @@
+support server workloads.
+
+Minimal
+~~~~~~~
+
+By providing a greatly reduced package set to minimise the security cross-section of the image, Ubuntu Minimal requires
+fewer security updates over time.
+
+CIS
+~~~
+
+A hardened Ubuntu image (see `Image Hardening`_), pre-configured to align with industry best practices and guidelines
+set by the Center for Internet Security (CIS). Ubuntu CIS images on Azure use custom tailoring files to meet both CIS
+benchmarks as well as ensure the image remains functional in Azure cloud environments.
+
+FIPS
+~~~~
+
+The Ubuntu FIPS image contains a Ubuntu kernel and security components which have been certified by the National
+Institute of Standards and Technology (NIST), making it a compelling choice for users operating in regulated
+environments such as FedRAMP, HIPAA, or PCI.
+
+Confidential VM
+~~~~~~~~~~~~~~~
+
+Supports Azure Confidential Virtual Machines, providing enhanced security features to protect data at rest, in use, and
+during boot.
+
+Ubuntu Pro
+----------
+
+Ubuntu Pro extends the capabilities of Ubuntu Server by incorporating advanced security, compliance features, and
+systems management tooling. It is designed for production environments where additional security layers and
+compliance with regulatory standards are paramount. Ubuntu Pro includes:
+
+* Expanded Security Maintenance: Security updates and patches for a wide array of applications and packages beyond the
+  standard offerings.
+* Kernel Livepatch: Minimises downtime and disruption by providing kernel updates that can be applied without rebooting
+  the system.
+* Up to 12 years of support: Long-term support that ensures your systems are secure and stable for more than a decade,
+  with a guaranteed upgrade path.
+
+To learn more about how to get Ubuntu Pro, see :doc:`../azure-how-to/instances/get-ubuntu-pro`.
+
+Image Hardening
+---------------
+
+The security profile for Ubuntu images on Azure can be further enhanced with `hardening <https://ubuntu.com/blog/what-is-system-hardening-definition-and-best-practices>`_
+by applying CIS hardening profiles from the `Ubuntu Security Guide (USG) <https://ubuntu.com/security/certifications/docs/usg>`_.
+
+Updates
+-------
+
+Latest Images
+~~~~~~~~~~~~~
+
+Ubuntu images for a given Azure Marketplace offer (see :doc:`../azure-how-to/instances/find-ubuntu-images`) are
+regularly published to deliver:
+
+* The latest kernel available for the image type.
+* Security patches to address CVEs.
+* The latest stable package set from the Ubuntu archives.
+
+Existing Workloads
+~~~~~~~~~~~~~~~~~~
+
+Deployments based off older image versions can receive the same security updates as the latest images by:
+
+* Keeping installed packages up to date with :code:`apt`.
+* Restarting the virtual machine to update the kernel, or leverage `Livepatch with Ubuntu Pro <https://ubuntu.com/security/livepatch>`_
+  to automatically update the kernel while the system is running.
```

Comment:
Once again, this is not an how-to but just documenting how to know if a system needs a reboot would be good here (check if `/var/run/reboot-required` exists).

Reply:
As [stated[(https://github.com/canonical/ubuntu-cloud-docs/pull/269#discussion_r1731955841), if the expectation is that every single cloud has a Security Overview document, all repeating `/var/run/reboot-required` - then this is a fundamental Ubuntu Security feature - not a cloud security feature and should be consolidated as part of Ubuntu Security resources (which I will admit are nascent and need considerable contributions to reach the level of detail that they should).